apps/OboeTester/app/src/main/cpp/analyzer/BaseSineAnalyzer.h

```diff
@@ -195,7 +213,9 @@ class BaseSineAnalyzer : public LoopbackProcessor {
     double  mOutputAmplitude = 0.75;
     // If this jumps around then we are probably just hearing noise.
     double  mPhaseOffset = 0.0;
+    static constexpr double kPhaseInvalid = -999.0;
```

Comment:
Can you add some comments on how this is used and when to use this?

Reply:
Done.